src/main/java/org/jitsi/videobridge/VideoChannel.java

```diff
@@ -323,6 +324,7 @@ void initialize(RTPLevelRelayType rtpLevelRelayType)
         throws IOException
     {
         super.initialize(rtpLevelRelayType);
+        bitrateController.update(null, -1);
```

Comment:
When does this happen in the flow? What case does this help with?

Reply:
https://github.com/jitsi/jitsi-videobridge/commit/e25cd5ef5aa9535a826e062cf5595a60e004b29b